### PR TITLE
new branch

### PR DESCRIPTION
**Pull request by dawidmiechowski**

### CODE REVIEW

**Comment by Sato2023** [comment]
hei

File: project1/app/views/pages/selectt.html.liquid [hunk 1]
```diff
@@ -51,9 +53,9 @@
             }
 
             
-            document.getElementById("toggleVisibilityButton").addEventListener("click", function(button) {
```
Comment (godot) [inline project1/app/views/pages/selectt.html.liquid:54]: please move all JS code into a separate file 
`app/assets/scripts/app.js`

**Review by godot — CHANGES_REQUESTED** [review]

**Comment by godot** [comment]
also please change a directory structure of the project
- delete nesting in a `project1` folder - meaning move everything from this folder up

**Review by godot — APPROVED** [review]
OK merge

**Pull request merged**